docs/src/intro/use_cases.md

```diff
@@ -40,9 +40,8 @@ model results were already quite satisfying, especially given the complex nature
 basins, including several small and big reservoirs. We could also demonstrate the potential
 for further calibration by changing the KsatHorFrac parameter.
 
-Reference: De Kleermaeker, S., Maguire, S., Druery, C., Morales, Y., Lijnse, T., Hegnauer, M.,
-2022 (in preperation). Developing a real-time data and modeling framework for operational
-flood inundation forecasting in Australia. Hydrology Water and Resources Symposium.
+Reference: De Kleermaeker, S., Leijnse, T., Morales, Y., Druery, C., Maguire, S.,
+2022. Developing a real-time data and modelling framework for operational flood inundation forecasting in Australia. In Hydrology & Water Resources Symposium 2022 (HWRS 2022): The Past, the Present, the Future. Engineers Australia. https://search.informit.org/doi/10.3316/informit.916755150845355
```

Comment:
Would be good to wrap these lines (e.g. with Rewrap (`Wrapping Column` of 92)).

Reply:
Done